cases/arithmetic/acosh.yaml

```diff
@@ -1,14 +1,5 @@
 function: acosh
 cases:
-  - group:
-      id: basic
-      description: Basic examples without any special cases
-    args:
-      - value: 0.01
-        type: fp32
-    result:
-      value: null
-      type: fp32
```

Comment:
why is this removed?

Reply:
Domain of `arcosh` function is [1,Infinity), so I added this case in the on_domain_error group.